docs/src/index.md

```diff
@@ -0,0 +1,18 @@
+# ChainRulesTestUtils
+
+[![Travis](https://travis-ci.org/JuliaDiff/ChainRulesTestUtils.jl.svg?branch=master)](https://travis-ci.org/JuliaDiff/ChainRulesTestUtils.jl)
+[![Code Style: Blue](https://img.shields.io/badge/code%20style-blue-4495d1.svg)](https://github.com/invenia/BlueStyle)
+
+
+[ChainRulesTestUtils.jl](https://github.com/JuliaDiff/ChainRulesTestUtils.jl) helps you test [`ChainRulesCore.frule`](http://www.juliadiff.org/ChainRulesCore.jl/dev/api.html) and [`ChainRulesCore.rrule`](http://www.juliadiff.org/ChainRulesCore.jl/dev/api.html) methods, when adding rules for your functions in your own packages.
```

Comment:
```suggestion
[ChainRulesTestUtils.jl](https://github.com/JuliaDiff/ChainRulesTestUtils.jl) helps you test [`ChainRulesCore.frule`]http://www.juliadiff.org/ChainRulesCore.jl/stable/api.html#ChainRulesCore.frule-Tuple{Any,Vararg{Any,N}%20where%20N}) and [`ChainRulesCore.rrule`]http://www.juliadiff.org/ChainRulesCore.jl/stable/api.html#ChainRulesCore.rrule-Tuple{Any,Vararg{Any,N}%20where%20N}) methods, when adding rules for your functions in your own packages.
```

Reply:
I purposefully didn't link the exact path, as it's too liable to change, whereas frule and rrule are likely to always be easy to find on that page